language-server/modules/langserver-core/src/test/resources/completion/worker/workerChannelReceive.json

```diff
@@ -0,0 +1,16 @@
+{
+    "position": {
+        "line": 7,
+        "character": 19
+    },
+    "source": "worker/source/workerChannelInteraction.bal",
+    "items": [
+        {
```

Comment:
Is this the only item we are getting in this particular scenario? If isn't shall we add all the returned items?

Reply:
yes, only `chn` will be sufficient for this scenario IMO.